sample/src/main/java/com/canhub/cropper/sample/SMainActivity.kt

```diff
@@ -12,16 +14,19 @@ import com.example.croppersample.databinding.ActivityMainBinding
 
 internal class SMainActivity : AppCompatActivity() {
 
+    private lateinit var binding: ActivityMainBinding
+
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
-        val binding = ActivityMainBinding.inflate(layoutInflater)
+        binding = ActivityMainBinding.inflate(layoutInflater)
```

Comment:
Why this change? We now have a variable that can be forgotten to init or don't get init in some future concurrency issue. 

I would prefer to keep it safe, since we can do in this case. What you think?